unified_targets/docs/Manufacturers.md

```diff
@@ -26,4 +26,5 @@ This is the official list of manufacturer ids (`manufacturer_id` in the target c
 |RUSH|FPV Racing Rush|http://www.rushfpv.com/|
 |SPRO|Seriously Pro Racing (SP Racing)|http://seriouslypro.com/|
 |TTRH|TransTEC|http://www.transtechobby.com/|
-|TCMM|TCMM|http://www.tcmm-rc.cn/|
\ No newline at end of file
+|TCMM|TCMM|http://www.tcmm-rc.cn/|
+|VIVA|VivaFPV|https://vivafpv.com/|
```

Comment:
That URL looks like an empty demo page - is there an URL that that users can actually use to buy their hardware, or get information about it?

Reply:
Here is where you can buy it (I have that included in the board location) however the above URL is the manufacturer (not where you can buy it). I anticipate they will populate that URL shortly.

https://www.team-blacksheep.com/products/prod:vivafc_esc_stacks

Reply:
Should we list them under Team Black Sheep until then?

Reply:
There will be others carrying the stock though... I have included TBS as a retailer in the "board - VIVAF4AIO.md" and will add others as they come up. I'll chase them for updating their website.

Reply:
Good stuff, https://vivafpv.com/ has come alive now!